packages/expo-device/src/ExpoDevice.web.ts

```diff
@@ -25,7 +25,15 @@ export default {
   get deviceYearClass(): null {
     return null;
   },
-  get totalMemory(): null {
+  get totalMemory(): number | null {
+    if ('deviceMemory' in navigator) {
+      // @ts-ignore
+      const { deviceMemory = 0 } = navigator;
+      const mb = deviceMemory * 1000;
+      const kb = mb * 1000;
+      const b = kb * 1000;
+      return b;
```

Comment:
The spec says GiB, not GB. Add a regression test so this error doesn't get added back accidentally.
```suggestion
      return deviceMemory * 1024 ** 3;
```